lib/compress.js

```diff
@@ -1731,6 +1731,7 @@ merge(Compressor.prototype, {
                 var val = {};
                 for (var i = 0, len = this.properties.length; i < len; i++) {
                     var prop = this.properties[i];
+                    if (prop instanceof AST_Expansion) return this;
```

Comment:
@alexlamsl Adding the following patch to `may_throw_on_access` does not fix the problem:
```js
        def(AST_Expansion, return_true);
```
What do you recommend?